### ISSUE TITLE
Overlapping lines when form has long title / description

### ISSUE THREAD

**Issue by gdei** [nextcloud/forms#43]
### Steps to reproduce
1. Create a form with a long title or description. Make it so long the resulting text won't fit on a single line
2. Open the form's response page

### Expected behaviour
The lines containing the title should be aligned neatly without overlap

### Actual behaviour
Title lines overlap

## Server configuration detail

**Operating system:** Linux 4.15.0-55-generic #60-Ubuntu SMP Tue Jul 2 18:22:20 UTC 2019 x86_64

**Webserver:** Apache (fpm-fcgi)

**Database:** mysql 5.7.27

**PHP version:** 

7.2.20
Modules loaded: Core, date, libxml, openssl, pcre, sqlite3, zlib, bz2, ctype, curl, dom, hash, fileinfo, filter, ftp, gd, SPL, iconv, intl, json, ldap, mbstring, pcntl, PDO, session, pdo_sqlite, posix, Reflection, standard, SimpleXML, mysqlnd, exif, tokenizer, xml, xmlreader, xmlwriter, zip, pdo_mysql, cgi-fcgi, redis, Zend OPcache

**Nextcloud version:** 15.0.10 - 15.0.10.0

**Updated from an older Nextcloud/ownCloud or fresh install:** fresh install

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.1.0
 - activity: 2.8.2
 - admin_audit: 1.5.0
 - announcementcenter: 3.4.1
 - apporder: 0.6.0
 - checksum: 0.4.3
 - cloud_federation_api: 0.1.0
 - comments: 1.5.0
 - dav: 1.8.2
 - deck: 0.6.5
 - federatedfilesharing: 1.5.0
 - federation: 1.5.0
 - files: 1.10.0
 - files_accesscontrol: 1.5.0
 - files_automatedtagging: 1.5.0
 - files_mindmap: 0.0.13
 - files_pdfviewer: 1.4.0
 - files_retention: 1.4.2
 - files_sharing: 1.7.0
 - files_texteditor: 2.7.0
 - files_trashbin: 1.5.0
 - files_versions: 1.8.0
 - files_videoplayer: 1.4.0
 - firstrunwizard: 2.4.0
 - forms: 1.0.3
 - fulltextsearch: 1.2.10
 - fulltextsearch_elasticsearch: 1.2.6
 - gallery: 18.2.0
 - groupfolders: 3.0.2
 - groupquota: 0.1.0
 - issuetemplate: 0.5.0
 - keeporsweep: 0.2.1
 - logreader: 2.0.0
 - lookup_server_connector: 1.3.0
 - nextcloud_announcements: 1.4.0
 - notifications: 2.3.0
 - oauth2: 1.3.0
 - password_policy: 1.5.0
 - polls: 0.10.2
 - provisioning_api: 1.5.0
 - quota_warning: 1.4.0
 - richdocuments: 3.3.14
 - serverinfo: 1.5.0
 - sharebymail: 1.5.0
 - spreed: 5.0.4
 - support: 1.0.0
 - survey_client: 1.3.0
 - systemtags: 1.5.0
 - terms_of_service: 1.1.2
 - twofactor_backupcodes: 1.4.1
 - user_ldap: 1.5.0
 - user_usage_report: 1.1.2
 - workflowengine: 1.5.0
Disabled:
 - encryption
 - files_external
 - theming

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "apps_paths": [
        {
            "path": "\/snap\/nextcloud\/current\/htdocs\/apps",
            "url": "\/apps",
            "writable": false
        },
        {
            "path": "\/var\/snap\/nextcloud\/current\/nextcloud\/extra-apps",
            "url": "\/extra-apps",
            "writable": true
        }
    ],
    "supportedDatabases": [
        "mysql"
    ],
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "memcache.local": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 0
    },
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "nextcloud.bamberg.wienet.ads"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "15.0.10.0",
    "overwrite.cli.url": "http:\/\/nextcloud.bamberg.wienet.ads",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "ldapIgnoreNamingRules": false,
    "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_sendmailmode": "smtp",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "25",
    "loglevel": 2,
    "maintenance": false
}
```
</details>

**Are you using external storage, if yes which one:** none

**Are you using encryption:** false

**Are you using an external user-backend, if yes which one:** LDAP

<details><summary>LDAP configuration (delete this par if not used)</summary>

</details>

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:60.0) Gecko/20100101 Firefox/60.0

**Operating system:** Windows 10

## Logs

<details><summary>Web server error log</summary>

```
Insert your web server log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

**Comment by marcelklehr** [comment]
Part of this is fixed by #41 (it avoids putting the description into the title), but you can still set a really long title to observe this.

**Comment by jotoeri** [comment]
Should be fixed now... (even if not really beautiful)

![grafik](https://user-images.githubusercontent.com/47433654/75663674-80315d00-5c71-11ea-8520-2e5e51121fca.png)
